doc/usermanual/lua/lua_api.xml

```diff
@@ -3551,19 +3551,20 @@ NOTE2: If the parameter is a directory the call is non-blocking; the film object
 </variablelist>
 </section>
 
-<section status="final" id="darktable_database_move_image">
+<section status="draft" id="darktable_database_move_image">
```

Comment:
Why using "draft" here ?

Reply:
I thought that was the procedure for any unreviewed section.